package.json

```diff
@@ -17,9 +17,9 @@
     "url": "https://github.com/su-sws/decanter.git"
   },
   "version": "0.1.0",
-  "dependencies": {
-    "bourbon": "~5.0.0-beta.7",
-    "bourbon-neat": "^2.0.0-beta.2",
-    "font-awesome": "^4.6.3"
+  "devDependencies": {
+    "bourbon": "5.0.0-beta.7",
+    "bourbon-neat": "2.0.0-beta.2",
```

Comment:
i'm specifying 5.0.0-beta.7 and 2.0.0-beta.2 without any `~` `>` `^` `=` modifiers since we want to be very deliberate about what might get pulled in. Font awesome I want to auto update, but it will still be good to come back and update as the package continues to evolve.